logRootQA.py

```diff
@@ -319,7 +319,8 @@ def summaryComp(compDir):
 for r in commonRoots:
 #    print 'I could have tested',r
     if ('PU' in r or 'RECODR' in r or 'REMINIAOD' in r) and 'DQM' not in r:
-        sameEvts=sameEvts and checkEventContent(baseDir+r,testDir+r)
```

Comment:
@davidlange6 
(as original developer)
was it intentional to have a check only until the first file difference?
As written, after `sameEvts` is False, the following statements are not evaluated.